Explorer/Assets/DCL/SDKComponents/NftShape/Component/NftLoadingComponent.cs

```diff
@@ -0,0 +1,27 @@
+using ECS.StreamableLoading;
+using Promise = ECS.StreamableLoading.Common.AssetPromise<UnityEngine.Texture2D, ECS.StreamableLoading.NftShapes.GetNftShapeIntention>;
+
+namespace DCL.SDKComponents.NftShape.Component
+{
+    public struct NftLoadingComponent
+    {
+        public Promise promise;
```

Comment:
It's mentioned: either in Rider's config or .editorconfig. We use both because nto everything can be convered by `.editorconfig`.
But if you have some local settings they override shared ones